src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -561,8 +561,11 @@ uct_cuda_ipc_query_devices(
         uct_md_h md, uct_tl_device_resource_t **tl_devices_p,
         unsigned *num_tl_devices_p)
 {
+    uct_cuda_ipc_md_t *ipc_md = ucs_derived_of(md, uct_cuda_ipc_md_t);
 #if HAVE_CUDA_FABRIC
-    uct_device_type_t dev_type = UCT_DEVICE_TYPE_NET;
+    uct_device_type_t dev_type =
+        ((ipc_md->enable_mnnvl == UCS_TRY) || (ipc_md->enable_mnnvl == UCS_YES))
```

Comment:
why needed?
seems it does not really affect VMM memory allocation/usage in cuda_ipc flows..

Reply:
@yosefe The motivation behind `enable_mnnvl` was to address [this ](https://github.com/openucx/ucx/pull/9982#discussion_r1671823426) request. 
So while this variable doesn't affect VMM allocation/usage, it does result in disabling MNNVL reachability. I've removed these changes for this PR and will address this separately.